tools/ci_build/github/azure-pipelines/nodejs/templates/test_linux.yml

```diff
@@ -1,19 +1,21 @@
 parameters:
   AgentPool: 'Linux-CPU'
   JobSuffix: ''
-
-jobs:
-- job: Nodejs_Test_${{ parameters.JobSuffix }}
-  workspace:
-    clean: all
-  timeoutInMinutes:  120
-  pool: ${{ parameters.AgentPool }}
+stages:
+- stage: Nodejs_Test_${{ parameters.JobSuffix }}
```

Comment:
nit: also can rename "JobSuffix" to "StageSuffix", here and in other build definitions